src/server/generic_family.cc

```diff
@@ -463,6 +463,33 @@ void GenericFamily::Stick(CmdArgList args, ConnectionContext* cntx) {
   (*cntx)->SendLong(match_cnt);
 }
 
+void GenericFamily::Move(CmdArgList args, ConnectionContext* cntx) {
+  string_view key = ArgS(args, 1);
+  int64_t target_db;
+
+  if (!absl::SimpleAtoi(ArgS(args, 2), &target_db)) {
+    return (*cntx)->SendError(kInvalidIntErr);
+  }
+
+  if (target_db < 0 || target_db >= absl::GetFlag(FLAGS_dbnum)) {
+    return (*cntx)->SendError(kDbIndOutOfRangeErr);
+  }
+
+  OpStatus res = OpStatus::SKIPPED;
+  ShardId target_shard = Shard(key, shard_set->size());
+  auto cb = [&](Transaction* t, EngineShard* shard) {
+    // MOVE runs as a global transaction and is therefore scheduled on every shard.
+    if (target_shard == shard->shard_id()) {
+      res = OpMove(t->GetOpArgs(shard), key, target_db);
+    }
+    return OpStatus::OK;
+  };
```

Comment:
Somewhat like a manual ScheduleSingleHopT. A bit tricky, but I didn't find a better solution.

Alternatively, we could make the transaction base not ignore keys on global transactions. If there are none, that receive arguments - then by default if they're not empty. Otherwise mark with some special flag like `GLOBAL_TRANS_WITH_KEYS`.

But that may be not worth it, if we'll support multi-db transactions

Reply:
i think it's fine for now.